pkg/controller/controller_utils_test.go

```diff
@@ -545,20 +642,23 @@ func TestSortingActivePodsWithRanks(t *testing.T) {
 		{p1: unscheduled5Hours, p2: unscheduled8Hours},
 		{p1: ready5Hours, p2: ready10Hours},
 	}
-	for _, tc := range equalityTests {
-		defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.LogarithmicScaleDown, !tc.disableLogarithmicScaleDown)()
-		if tc.p2 == nil {
-			tc.p2 = tc.p1
-		}
-		podsWithRanks := ActivePodsWithRanks{
-			Pods: []*v1.Pod{tc.p1, tc.p2},
-			Rank: []int{1, 1},
-			Now:  now,
-		}
-		if podsWithRanks.Less(0, 1) || podsWithRanks.Less(1, 0) {
-			t.Errorf("expected pod %q to be equivalent to %q", tc.p1.Name, tc.p2.Name)
+	t.Run("Equality tests", func(t *testing.T) {
```

Comment:
Please also here revert the order of `t.Run` and iteration over test cases. Then, name the test cases based on their indexes `fmt.Sprintf("test%d", i)`

Reply:
As with the inequality tests, I'll name these "Equality tests %d" to give a bit more context